spec/controllers/webhook_events_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe WebhookEventsController, type: :controller do
+  include ActiveJob::TestHelper
+
+  let(:organization)  { GitHubFactory.create_owner_classroom_org }
+  let(:user)          { organization.users.first                 }
+
+  describe 'POST #create' do
+    describe 'ping event' do
+      context 'valid payload signature' do
+        before(:each) do
+          params = JSON.parse(File.open("#{Rails.root}/spec/support/fixtures/webhook_payloads/ping.json").read)
+          params.symbolize_keys!
+          params[:sender][:id] = user.uid
+          params[:organization][:id] = organization.github_id
```

Comment:
I modify the params manually since we need to verify the existences of the `sender` and the `organization`.